lib/gs1/extensions/gtin.rb

```diff
@@ -0,0 +1,30 @@
+module GS1
+  module Extensions
+    # Extension for a GS1 GTIN. Ensures correct formating and validation.
+    #
+    module GTIN
+      def self.included(base)
+        base.extend         LengthValidation::ClassMethods
+        base.send :include, LengthValidation::InstanceMethods
+        base.send :include, CheckDigitValidation
+
+        base.valid_lengths [8, 12, 13, 14].freeze
```

Comment:
These lengths does not match `VALID_LENGTHS`  in `CheckDigitValidation`, is it intended?

Reply:
`CheckDigitCalculator` validates the input sequence without the check digit, thus `VALID_LENGTHS - 1`